samples/kubernetes/end2end/docs/03-wls-operator.md

```diff
@@ -0,0 +1,77 @@
+## Installing the WLS Operator
+We depend on [the WLS operator](https://github.com/oracle/weblogic-kubernetes-operator) to create and manage WLS domains. Check [the installation doc](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-operators/installation/).
+
+Here we provide the detail steps to install the WLS operator with release 2.1 as a example.
+
+
+Pull and tag docker images. **TODO: use OCR store?**
+```
+docker pull wlsldi-v2.docker.oraclecorp.com/store-weblogic-12.2.1.3:latest
+docker tag wlsldi-v2.docker.oraclecorp.com/store-weblogic-12.2.1.3:latest store/oracle/weblogic:12.2.1.3
+docker pull oracle/weblogic-kubernetes-operator:2.1
+```
+Verify that the weblogic image has the right patch set.
+```
+docker run store/oracle/weblogic:12.2.1.3 sh -c '$ORACLE_HOME/OPatch/opatch lspatches'
```

Comment:
The image should come from OCR container-registry.oracle.com/middleware/weblogic:12.2.1.3